pkg/streamingpromql/testdata/ours/functions.test

```diff
@@ -436,3 +436,21 @@ eval range from 0 to 54m step 6m clamp_min(series, scalar(mins))
 
 eval range from 0 to 54m step 6m clamp_max(series, scalar(maxes))
   {}     0 _ 0 0 5 _ 3 NaN 2 NaN
+
+clear
+
+load 6m
+  toNearest      1    _    3 4 0.1
+  series{a="b"}  -5.5 2.75 _ _ 6.87
+
+eval range from 0 to 24m step 6m round(series, scalar(toNearest))
+  series{a="b"}  -5   NaN  _ _ 6.9
+
+clear
+
+load 6m
+  toNearest      1    2     0.5   10    0.1   0.25    100       0.0001  5          1000      NaN _   _   NaN _ 2 2
```

Comment:
What happens if the argument is negative?

Reply:
Added test. (Basically the way round is implemented makes it take the absolute value)